libmscore/element.h

```diff
@@ -234,7 +235,7 @@ class Element : public QObject, public ScoreElement {
             SYSTEM_DIVIDER,
             STEM_SLASH,
             LINE,
-
+            ANNOTATION,
             ARPEGGIO,
             ACCIDENTAL,
```

Comment:
Probably warrants a comment on both places of the source that these 2 need to be kep in sync